.github/workflows/release_js_project.yaml

```diff
@@ -42,3 +42,29 @@ jobs:
       branch: releases/v${{ needs.extractChangelog.outputs.version }}
       version: v${{ needs.extractChangelog.outputs.version }}
       body: ${{ needs.extractChangelog.outputs.body }}
+  updateMajorTag:
+     runs-on: ubuntu-latest
+     permissions:
+       contents: write
+     needs: [extractChangelog, createRelease]
+     steps:
+       - uses: actions/checkout@v4
+       - name: get major version tag
+         id: major-version-tag
+         env:
+           NEW_RELEASE_TAG: ${{ needs.extractChangelog.outputs.version }}
+         run:
+           echo "MAJOR_VERSION_TAG=v${NEW_RELEASE_TAG%%.*}" >> "$GITHUB_OUTPUT"
+       - name: delete old major tag
+         continue-on-error: true
+         env:
+            GH_TOKEN: ${{ github.token }}
+            MAJOR_VERSION_TAG: ${{ steps.major-version-tag.outputs.MAJOR_VERSION_TAG }}
+         run:
+           gh release delete --cleanup-tag -y $MAJOR_VERSION_TAG
+       - name: create new major tag
+         env:
+            GH_TOKEN: ${{ github.token }}
+            MAJOR_VERSION_TAG: ${{ steps.major-version-tag.outputs.MAJOR_VERSION_TAG }}
+         run:
+           gh release create $MAJOR_VERSION_TAG --target releases/v${{ needs.extractChangelog.outputs.version }} -n "Latest $MAJOR_VERSION_TAG Release"
```

Comment:
should we create release or just create tag? I'm of the opinion that we should only create a tag

Reply:
that's a good point we can switch to tag-only